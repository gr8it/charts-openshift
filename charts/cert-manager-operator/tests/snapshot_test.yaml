suite: Snapshot test
templates:
  - charts/*/templates/*
release:
  name: cert-manager-operator
  namespace: apc-cert-manager-operator

tests:
  - it: manifests should match snapshot
    asserts:
      - matchSnapshot: {}
