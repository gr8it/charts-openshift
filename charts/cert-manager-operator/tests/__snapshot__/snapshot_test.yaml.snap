manifests should match snapshot:
  1: |
    apiVersion: policy.open-cluster-management.io/v1beta1
    kind: OperatorPolicy
    metadata:
      labels:
        app.kubernetes.io/instance: cert-manager-operator
        app.kubernetes.io/managed-by: Helm
        app.kubernetes.io/name: acm-operatorpolicy
        app.kubernetes.io/version: v1beta1
        helm.sh/chart: acm-operatorpolicy-1.1.0
      name: cert-manager-operator-acm-operatorpolicy
      namespace: open-cluster-management-policies
    spec:
      complianceConfig:
        catalogSourceUnhealthy: Compliant
        deploymentsUnavailable: NonCompliant
        upgradesAvailable: Compliant
      complianceType: musthave
      remediationAction: enforce
      removalBehavior:
        clusterServiceVersions: Delete
        customResourceDefinitions: Keep
        operatorGroups: DeleteIfUnused
        subscriptions: Delete
      severity: medium
      subscription:
        channel: stable-v1
        name: openshift-cert-manager-operator
        source: redhat-operators
        sourceNamespace: openshift-marketplace
        startingCSV: cert-manager-operator.v1.15.1
      upgradeApproval: Automatic
      versions:
        - cert-manager-operator.v1.15.1
